products/mssqlserver.md

```diff
@@ -11,135 +11,408 @@ versionCommand: select @@version
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
 
 activeSupportColumn: true
+releaseColumn: true
 releaseDateColumn: true
 extendedSupportColumn: Extended Security Updates
 
+# releaseDate/support/eol/extendedSupport: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
+# Overview over all versions: https://learn.microsoft.com/troubleshoot/sql/releases/download-and-install-latest-updates
 releases:
 -   releaseCycle: "2022"
```

Comment:
Should we use 16 as the release cycles, and 2022 as the release label ?

Reply:
Update done see f093bdaeb242264dd7445e3b3ec555d31253deac.